genshin/models/hoyolab/record.py

```diff
@@ -110,8 +110,8 @@ class HoyolabUserCertification(APIModel):
     For example artist's type is 2.
     """
 
-    icon_url: str
-    description: str = Aliased("desc")
+    icon_url: typing.Optional[str]
+    description: typing.Optional[str] = Aliased("desc")
```

Comment:
Should have explicit ` = None` for compatibility with older pydantic versions

Reply:
Fixed.